kamon-statsd/src/main/resources/reference.conf

```diff
@@ -5,18 +5,10 @@
 kamon {
   statsd {
 
-    # Hostname and port in which your StatsD is running. Remember that StatsD packets are sent using UDP and
-    # setting unreachable hosts and/or not open ports wont be warned by the Kamon, your data wont go anywhere.
-    hostname = "127.0.0.1"
-    port = 8125
```

Comment:
@ivantopo I agreed with you, I think  that is a good idea keep it simple for most users and i prefer use the old `kamon.statsd.[hostname|port]`